docs/contributing.md

```diff
@@ -57,7 +57,7 @@ Never made an open source contribution before? Wondering how contributions work
 Most of our style is pretty easy to pick up on, but right now it's not entirely consistent. You should match the style of the code surrounding your change, but if that code is inconsistent or unclear use the following guidelines:
 
 * We indent using two spaces (soft tabs)
-* We use One True Brace Style
+* We use a modified One True Brace Style
```

Comment:
I'd say it's a One True Brace Style variant rather than a modification. But I'm nitpicking here, feel free to ignore me.